tests/ttnn/unit_tests/operations/backward/test_backward_rsub.py

```diff
@@ -29,3 +29,52 @@ def test_bw_rsub(input_shapes, device):
 
     status = compare_pcc(tt_output_tensor_on_device, golden_tensor)
     assert status
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+@pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True]])
+def test_bw_rsub_opt(input_shapes, device, are_required_outputs):
+    in_data, input_tensor = data_gen_with_range(input_shapes, -100, 100, device, True)
+    other_data, other_tensor = data_gen_with_range(input_shapes, -5, 5, device, True)
+
+    grad_data, grad_tensor = data_gen_with_range(input_shapes, -10, 5, device)
+
+    input_grad = None
+    other_grad = None
+    tt_output_tensor_on_device = None
```

Comment:
i this necessary?

Reply:
Since we pass it as a parameter, it must be defined first. We create optional tensors based on `are_required_outputs`, but they need to be defined beforehand, regardless of whether the value is `True` or `False`